<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task5</class>
 <widget class="QMainWindow" name="task5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1446</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1201</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxMax_y">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxMin_y">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>90</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>590</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;x range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;y range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>600</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;n&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>600</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Pn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>590</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pn(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Pn_Ln">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pn(x) − ln (x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>610</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View init row</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_L">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>590</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Лагранж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>40</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>погрешность</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_p">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>180</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_N">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>620</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ньютон</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1446</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutask5">
    <property name="title">
     <string>task5</string>
    </property>
   </widget>
   <addaction name="menutask5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
